src/helper/urls/get-auth-urls.ts

```diff
@@ -0,0 +1,9 @@
+// helpers while we test kratos alongside the old version
+
+export function getPasswordChangeUrl() {
```

Comment:
@hugotiburtino does this need to be a function?
Could we use a const string as well?

Reply:
Sure, I'll change it

Reply:
@elbotho , do you prefer hard coding it (like in other [menu strings](https://github.com/serlo/frontend/blob/staging/src/data/de/index.ts#L415)?)?